vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/select_autocomplete.tsx

```diff
@@ -68,9 +68,8 @@ export default function SelectAutocompleteInput(
     <Autocomplete
       disableClearable={disableClearable}
       clearOnBlur={true}
-      options={options.display}
-      // disable filtering on client
-      filterOptions={(x: string[]) => x}
+      options={options_in}
+      filterOptions={(x: string[]) => options.display}
```

Comment:
I never encountered it, but this change likely prevents similar MUI complaint of 'value not valid' in cases where the user selects a value that's not within the first 100/`maxOptions` options.